src/components/chatting/ChattingLeft.jsx

```diff
@@ -0,0 +1,12 @@
+import PublicChannelList from "./talkplus/PublicChannelList";
+import UserBox from "./talkplus/UserBox";
+
+const ChattingsLeft = ({ loginUserInfo }) => {
```

Comment:
```suggestion
const ChattingLeft = ({ loginUserInfo }) => {
```
파일명이랑 컴포넌트 명이 살짝 다르네요.ㅎㅎ

Reply:
수정하도록 하겠습니다!